Move /spin page to a new section on /icons page (#2247)
vanilla-framework/vanilla-framework: docs/en/patterns/icons.md

  </div>
</section>

### Animation

> Animate an element by spinning with the follow utility class.

Reviewer: ```suggestion
Add a spin animation to an element with the following utility class.
```
